ocs_ci/utility/utils.py

```diff
@@ -1137,3 +1137,27 @@ def ceph_health_check(namespace=None):
         raise CephHealthException(
             f"Ceph cluster health is not OK. Health: {health}"
         )
+
+
+def get_rook_repo(to_checkout):
+    """
+    Clone and checkout the rook repository to specific branch/commit.
+
+    Args:
+        to_checkout (str): Commit id or branch to checkout on
+
+    """
+    fresh_clone = False
+    cwd = constants.ROOK_REPO_DIR
+    if not os.path.isdir(constants.ROOK_REPO_DIR):
+        log.info("Cloning rook repository.")
```

Comment:
I would maybe include the `cwd` (`ROOK_REPO_DIR`) into the log message.

Reply:
I didn't put it here cause run_cmd is writing the command which is running to the logger. But added it here as well if you find it useful.